src/client.ts

```diff
@@ -62,12 +62,16 @@ export class SkynetClient {
   resolveHns = resolveHns;
 
   // SkyDB
-  getFile = getFile;
-  setFile = setFile;
+  db = {
```

Comment:
not a priority now I guess but this will not be typed properly in the SkynetClient I don't think

Reply:
easily fixed by the way, just define an interface 

```
export interface IDB {
  getJSON(
    publicKey: PublicKey,
    dataKey: string
  ): Promise<Record<string, unknown> | null>
  
  setJSON(
    privateKey: SecretKey,
    dataKey: string,
    json: Record<string, unknown>,
    revision?: number
  ): void;
}
```
and do 
```
  resolveHns = resolveHns;

  // SkyDB
  db: IDB = {
    getJSON: getJSON.bind(this),
  ...
```

(rename the interface, it's just for demonstrating purposes)

Reply:
Not so easily-fixed, I get typescript errors when I do this. I'm just going to make a follow-up issue for later, thanks.

Reply:
https://github.com/NebulousLabs/skynet-js/issues/144